.env.example

```diff
@@ -42,6 +42,15 @@ MAIL_FROM_NAME=
 MAILGUN_DOMAIN=
 MAILGUN_SECRET=
 
+# Slack Webhook Endpoint path to be appended to https://hooks.slack.com/services/.
+SLACK_ENDPOINT=
+
+# Default slack channel for the bot to post in.
+SLACK_CHANNEL=
+
+# Default username to post as
+SLACK_USERNAME=
```

Comment:
Could it be `SLACK_ENDPOINT_USERNAME`?

This stops us confusing it with the actual BOT username.